### PR TITLE
feat(Core/DBUpdater): Apply each sql update file in a separate transa…

### PR DESCRIPTION
**Pull request by Nyeriah**
…ction

## Changes Proposed:
-  Apply each SQL file as a separate transation.

### CODE REVIEW

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
Reference the original commit too please 💚

**Comment by Nyeriah** [comment]
Feel free to add it to the merge commit message although it creates useless GitHub notifications that may or may not be desired

File: src/server/database/Updater/DBUpdater.cpp [hunk 1]
```diff
@@ -491,13 +491,17 @@ void DBUpdater<T>::ApplyFile(DatabaseWorkerPool<T>& pool, std::string const& hos
 
 #endif
 
+    // Execute sql file
+    args.emplace_back("-e");
+    args.emplace_back(Acore::StringFormat("BEGIN; SOURCE %s; COMMIT;", path.generic_string().c_str()));
```
Comment (Winfidonarleyan) [inline src/server/database/Updater/DBUpdater.cpp:496]: StringFormatFmt

**Comment by Kitzunu** [comment]
> Feel free to add it to the merge commit message although it creates useless GitHub notifications that may or may not be desired

I mean not to @ but just to link the commit hash from TC, that shouldnt ping no?

**Comment by Nyeriah** [comment]
Ingen aning 🤷‍♂️

**Pull request merged**